libxw/pyproject.toml

```diff
@@ -0,0 +1,11 @@
+# Configuration file for how to "build" our package.
+#
+# https://www.python.org/dev/peps/pep-0517/
+# https://setuptools.readthedocs.io/en/latest/build_meta.html
+
+[build-system]
+requires = [
+  "setuptools>=40.9.0",
```

Comment:
This [version of `setuptools`](https://setuptools.pypa.io/en/latest/history.html?highlight=changelog#v40-9-0) is the first one to support `setup.cfg`-only (i.e., no `setup.py` at all) projects.